component/vdr/orb/config/service.go

```diff
@@ -333,14 +333,85 @@ func (cs *Service) getEndpointIPNS(didURI string) (*models.Endpoint, error) {
 		return nil, fmt.Errorf("get anchor origin didn't return string")
 	}
 
-	if !strings.HasPrefix(anchorOrigin, "ipns://") {
-		return nil, fmt.Errorf("anchor origin is not ipns")
+	currentAnchorOrigin := anchorOrigin
+
+	for {
+		latestAnchorOrigin, errGet := cs.getLatestCID(currentAnchorOrigin, didSplit[4])
```

Comment:
function should be called getLatestAnchorOrigin

Reply:
done